core/util/core_util.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <cuda_runtime.h>
+#include <string>
+
+namespace trtorch
+{
+namespace core
+{
+namespace util
+{
+    typedef struct {
```

Comment:
should we just make all the operators on this struct members of the struct?

Reply:
Are you referring to all the CudaDeviceProp attributes? We don't need those.

Reply:
No like the getters and setters, we should just make them methods of this struct. Also I dont think we need the typedef

Reply:
Added